libs/design/src/molecules/backdrop/backdrop/backdrop.component.scss

```diff
@@ -1,14 +1,16 @@
 @import '../../../scss/daff-util';
+$bg-color: rgba(0, 0, 0, 0.3);
+$highlight-color: rgba(0, 0, 0, 0);
 
 :host {
 	@include clickable();
 	display: block;
-	-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+	-webkit-tap-highlight-color: $highlight-color;
```

Comment:
I think this can be removed in general? `rgba(0, 0, 0, 0)` = `transparent`

Reply:
So remove the line all together? No highlight color needs to be specified?

Reply:
I think can be ignored. They are the same, but I think we prefer color functions (rgba) over color literalls (transparent).